CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.15.1)
+
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL Darwin)
+    execute_process(
+        COMMAND xcrun --sdk macosx --show-sdk-path
+        OUTPUT_VARIABLE CMAKE_OSX_SYSROOT
+        OUTPUT_STRIP_TRAILING_WHITESPACE)
+    SET(CMAKE_Swift_FLAGS "${CMAKE_Swift_FLAGS} -sdk ${CMAKE_OSX_SYSROOT}")
+endif()
+
+project(SwiftTSC LANGUAGES C Swift)
+ 
+set(SWIFT_VERSION 5)
```

Comment:
Why have a variable for this?  Why not just set it directly on L15?

Reply:
It's also used in L18